packages/core/src/index.ts

```diff
@@ -8,15 +8,22 @@ import {
   KB_JWT_TYP,
   SDJWTCompact,
   SDJWTConfig,
-  SD_JWT_TYP,
 } from '@sd-jwt/types';
 
 export * from './sdjwt';
 export * from './kbjwt';
 export * from './jwt';
 export * from './decoy';
 
-export class SDJwtInstance {
+export interface SDJwtPayload {
+  // more entries
+  [key: string]: unknown;
+}
```

Comment:
```suggestion
export type SdJwtPayload = Record<string, unknown>
```

Reply:
The `[key:string]: unknown` approach allows us to add variables later that should be / or are optional but defined here.

Reply:
E.g. we could set the 'MAY' referenced values here as optional, https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-07.html#section-5.1-2